CHANGELOG.rst

```diff
@@ -13,6 +13,12 @@
   bugs #965 and #967.
   (:issue:`643`, :pr:`657`, :user:`benjeffery`)
 
+**Breaking changes**:
```

Comment:
0.2.3 is already released, need a new release 0.2.4. Date = 2022-06-xx

Reply:
Oops, sorry. Changed.